### PR TITLE
[OMG READY] Ignite 2 On Final Approach

### PR DESCRIPTION
**Pull request by skellock**
We took a poll.  Should we merge v2 into v1 now or later?

Team now won.

Ignite 2 has been living in `next`.  This is compatible with `master` because v2 lives under `/packages/` and shares nothing in common.  This will cause some minor nuisances and confusion with contributors, so we should mention something on the README.

Maybe we could move `v1` out of the way sooner rather than later?

But tonight I say move forward, not backward... https://www.youtube.com/watch?v=zGmfiKowrYU

### CODE REVIEW

**Comment by GantMan** [comment]
dat commit log

**Comment by jamonholmgren** [comment]
I'd like to get this README shined up before we merge. Make sure people know where we are and where we aren't, how to use Ignite 2 vs 1, etc.

**Comment by skellock** [comment]
@GantMan I know right?  We kicked some ace.

@jamonholmgren I agree.  We should just wait before merging.  There's no reason yet.

File: packages/ignite-cli/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# ignite-cli
+
+Currently this is an unused.  But we may move here in the future if we decide to split
```
Comment (jamonholmgren) [inline packages/ignite-cli/readme.md]: > this is an unused.

unused what

**Comment by Un3qual** [comment]
Maybe add a `--v2` option to the generator for now so that people can decide which version to use until v2 has been out for a while.

**Comment by Un3qual** [comment]
Will `react-native` be updated from `38` to the latest (`42` I belive) in 2.0?

**Comment by skellock** [comment]
@Un3qual We'll be publishing under a different name: `ignite-cli`, so you can keep Ignite 1.  Ignite 1 will remain at 0.39.2.  Ignite 2 will be tracking the latest official release (currently 0.41.2).

**Comment by Un3qual** [comment]
Sounds good. Thanks!

**Pull request merged**